eng/pipelines/live-test-cleanup.yml

```diff
@@ -53,12 +53,46 @@ stages:
           -Verbose
         displayName: Clean up subscription resources (AzureChinaCloud)
 
+      - template: /eng/common/TestResources/build-test-resource-config.yml
+        parameters:
+          SubscriptionConfigurations:
+            - $(sub-config-azure-cloud-test-resources)
+            - $(sub-config-communication-services-cloud-test-resources)
       - pwsh: |
           $subscriptionConfiguration = @'
-            $(acs-subscription-configuration)
+            $(SubscriptionConfiguration)
           '@ | ConvertFrom-Json -AsHashtable
 
           ./tools/live-test-cleanup/cleanup.ps1 `
           @subscriptionConfiguration `
           -Verbose
-        displayName: Clean up subscription resources (AzureCloud ACS Subscription)
\ No newline at end of file
+        displayName: Clean up subscription resources (AzureCloud ACS Subscription)
+
+      - template: /eng/common/TestResources/build-test-resource-config.yml
+        parameters:
+          SubscriptionConfigurations:
+            - $(sub-config-communication-int-test-resources)
+      - pwsh: |
+          $subscriptionConfiguration = @'
+            $(SubscriptionConfiguration)
+          '@ | ConvertFrom-Json -AsHashtable
+
+          ./tools/live-test-cleanup/cleanup.ps1 `
+          @subscriptionConfiguration `
+          -Verbose
+        displayName: Clean up subscription resources (PreProduction ACS Subscription)
+
+      - template: /eng/common/TestResources/build-test-resource-config.yml
```

Comment:
I assume each time you include this it resets SubscriptionConfiguration variable?

Reply:
Yes. I thought about making the variable name dynamic as an option, but it seemed like overkill.